src/oaklib/implementations/ols/ols_implementation.py

```diff
@@ -73,6 +73,9 @@ def labels(self, curies: Iterable[CURIE]) -> Iterable[Tuple[CURIE, str]]:
     def annotate_text(self, text: str) -> Iterator[TextAnnotation]:
         raise NotImplementedError
 
+    def annotate_file(self, text: str) -> Iterator[TextAnnotation]:
```

Comment:
shouldn't be necessary as calling this will raise NotImplementedError via the default implementation... but not a big deal